### ISSUE TITLE
[Feature] Add learning rate scaler for elastic dataloader (e.g. AdaScale, Square-Root).

### ISSUE THREAD

**Issue by Antlera** [intelligent-machine-learning/dlrover#657]
Elastic dataloader are introduced to optimize batch size on the runtime. A scaler for learning rate is needed to ensure good training quality.

**Comment by xin-chen42** [comment]
I will provide a viable learning rate scaler by comparing similar issues.

**Comment by xin-chen42** [comment]
This issue has been resolved by PR #714.

**Comment by Antlera** [comment]
Closed as completed by #714.
